apis/v1alpha1/opentelemetrycollector_webhook.go

```diff
@@ -129,6 +129,10 @@ func (r *OpenTelemetryCollector) validateCRDSpec() error {
 			return fmt.Errorf("the OpenTelemetry Spec autoscale configuration is incorrect, minReplicas should be one or more")
 		}
 
+		if r.Spec.TargetCPUUtilization != nil && (*r.Spec.TargetCPUUtilization < int32(1) || *r.Spec.TargetCPUUtilization > int32(99)) {
```

Comment:
We should also be setting a default in this file

Reply:
Thanks for the review! Setting the default in opentelemetrycollector_webhook.go now